src/ocamlorg_frontend/pages/community.eml

```diff
@@ -37,27 +37,35 @@ Layout.render
         </a>
         <a href="https://github.com/ocaml/ocaml" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/github.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Github</div>
-          <div class="font-normal text-xs mt-4">Open bug reports and feature requests on the compiler</div>
+          <div class="font-normal text-xs mt-4">Open bug reports and feature requests on the OCaml compiler.</div>
         </a>
         <a href="https://discord.gg/cCYQbqN" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/discord.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Discord</div>
           <div class="font-normal text-xs mt-4">Chat with other members of the community on Discord.</div>
         </a>
         <a href="https://twitter.com/ocamllang" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/twitter.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Twitter</div>
-          <div class="font-normal text-xs mt-4">Catch up with some of the latest announcement and events from the community.</div>
+          <div class="font-normal text-xs mt-4">Catch up with some of the latest announcements and events from the community.</div>
+        </a>
+        <a href="https://types.pl/" class="border-gray-200 border p-8 rounded-xl card-hover">
+          <div class="text-base font-semibold"><img src="/img/community/mastodon.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Mastodon</div>
+          <div class="font-normal text-xs mt-4">Join the community on Mastodon, a decentralized social network. The types.pl Mastodon homeserver is popular among the OCaml community.</div>
         </a>
         <a href="irc://irc.libera.chat/#ocaml" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/hashtag.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> IRC Chat</div>
-          <div class="font-normal text-xs mt-4">Chat with other members of the community on IRC.</div>
+          <div class="font-normal text-xs mt-4">Chat with other members of the community at #ocaml on the Libera Chat IRC network.</div>
+        </a>
+        <a href="https://matrix.to/#/#ocaml-language:matrix.org" class="border-gray-200 border p-8 rounded-xl card-hover">
```

Comment:
Let's link to the OCaml space instead: https://matrix.to/#/#ocaml:roscidus.com -- that one adds other rooms in addition to the OCaml language chat one.